lib/ssl/src/ssl_certificate.erl

```diff
@@ -589,18 +588,33 @@ verify_cert_extensions(Cert, #{cert_ext := CertExts} =  UserState) ->
 
 verify_cert_extensions(Cert, UserState, [], _) ->
     {valid, UserState#{issuer => Cert}};
-verify_cert_extensions(Cert, #{ocsp_responder_certs := ResponderCerts,
-                               ocsp_state := OscpState,
-                               issuer := Issuer} = UserState,
-                       [#certificate_status{response = OcspResponsDer} | Exts],
+verify_cert_extensions(Cert, #{stapling_state := StaplingState,
```

Comment:
what if status_request_ext is negotiated but actual OCSP response is not received?
will code fail in such situation?

Reply:
It is handled by this code in  ssl_handshake.  (ocsp_expect := stapled means that the stapel information was recived)

cert_status_check(_,
                  #{ocsp_state := #{ocsp_stapling := true,
                                    ocsp_expect := stapled}},
                  _VerifyResult, _, _) ->
    %% OCSP staple will now be checked by
    %% ssl_certificate:verify_cert_extensions/2 in ssl_certificate:validate
    valid;

Reply:
yes. thank you.